Rename deviceIdClaim to match convention

Danielius1922 opened the pull request.

Danielius1922 committed:
Rename deviceIdClaim to match convention

jkralik reviewed (commented):
pls update go-coap/v2 to the master branch for build failed

Danielius1922 committed:
Upgrade dependencies

Direct:
github.com/plgd-dev/go-coap/v2 v2.5.1-0.20220202152015-d741687e6a37
github.com/plgd-dev/device v1.1.1-0.20220201194341-13db9d33afec

Indirect
golang.org/x/crypto v0.0.0-20220131195533-30dcbda58838

ondrejtomcik commented on resource-directory/service/getEventsSnapshot_test.go:
```diff
@@ -77,6 +79,7 @@ func TestRequestHandlerGetEventsStateSnapshot(t *testing.T) {
 			},
 		})
 		require.NoError(t, err)
+		time.Sleep(time.Millisecond * 200)
```
What has this to do with deviceIDClaim? :)
